templates/keepalived.conf.j2

```diff
@@ -93,6 +93,13 @@ vrrp_instance {{ name }} {
   {% if instance.dont_track_primary is defined and instance.dont_track_primary | bool %}
   dont_track_primary                               # Override VRRP RFC dont_track_primary default
   {% endif %}
+  {% if instance.accept is defined and instance.accept | bool %}
```

Comment:
```suggestion
  {% if instance.accept is defined %}
```

Reply:
This way if the user doesn't define accept (the default), there is no change in the config file.
And if someone writes instance.accept false, would write no_accept appropriately.

Reply:
I've not tested the alternative (nor do I have an objection to it) but I'd have thought that my if which is checking if accept is defined and a boolean wouldn't pass in the case of `accept: `, thus not writing out `accept` or `no_accept` to the file.